### ISSUE TITLE
Add Logitech Media Server

### ISSUE THREAD

**Issue by asche77** [basings/selfhosted-music-overview#31]
**Table Content**
"Logitech Media Server" (a.k.a. Squeezebox Server) (downloads via https://downloads.slimdevices.com/, information mainly on https://forums.slimdevices.com/forumdisplay.php?27-Logitech-Media-Server) should "check" the following:

<html><body>

Scrobble Plays | Can Read Tags | Can Write Tags | Subsonic API | Can Share Music | Multi-User Support | Multi-Library Support | Smart Playlists | Heart/ Favorites | 5 Star Rating | Replay Gain | Transcode | free | Demo | Source Code | License | Reviewed Version
✔️ | ✔️ | -- | -- | -- | (✔️) 1 | ✔️ 2 | ✔️ 3 | ✔️ | (✔️) 4 | ✔️ | ✔️ | ✔️ | -- | ✔️ | -- | 8.2

1 - not sure
2 - via plugin
3 - works best if music library has been analysed by MusicIP beforehand, otherwise limited capability
4 - not sure what this relates to, but can play "only 5 star rated songs"

</body>
</html>

**Comment by Jucgshu** [comment]
There's a JSON-RPC API available for this one. Also, what's the difference between LMS & Logitech Media Server? I've always assumed it was the same software? Aka Squeezebox?

Btw, the API documentation is available at http://SERVER-HOST:PORT/html/docs/cli-api.html?player=

**Comment by asche77** [comment]
Thought so myself, but apparently LMS stands for lightweight media server.

**Comment by billotronic** [comment]
Does Random Album and Songs not count as a jukebox mode?
https://imgur.com/Thhr7gD.png

**Comment by basings** [comment]
@billotronic I'll make it a :heavy_check_mark: =)

**Comment by epoupon** [comment]
@billotronic jukebox mode in the subsonic world means clients just queue songs and these songs are played on the server's audio output.

**Comment by asche77** [comment]
Hi

Scrobbling - yes. It needs a plugin but works fine (and the plugin is available out-of-the-box).
Multiroom -- yes sir!  Logitech Media Server was one of the first system offering synchronised multi-room audio.
API -- sorta kinda, so "custom". 
Lyrics -- a plugin is available but ... have not tested it and probably somewhat cumbersome.

**Comment by basings** [comment]
@asche77 thx! Do you know which scrobble service is supported? last.fm, listenbrainz, or another one?

**Comment by asche77** [comment]
The plugin is for last.fm (and works). 
It might be possible to modify the plugin / use a DNS redirect to use libre.fm instead, but I have not tested (and a quick Google did not throw up anything useful).

**Comment by asche77** [comment]
Another thing to note re squeezebox / API: it is not as widely supported as subsonic, but there are several 3rd party solutions that use/support it, e.g. non-exhaustive Room, Vortexbox, piCore/HiFiBerry, Daphile, Volumio, ...

**Comment by basings** [comment]
thanks! updated it.

**Comment by asche77** [comment]
> Another thing to note re squeezebox / API: it is not as widely supported as subsonic, but there are several 3rd party solutions that use/support it, e.g. non-exhaustive Roon, Vortexbox, piCore/HiFiBerry, Daphile, Volumio, ...

Adding https://moodeaudio.org/ which uses squeezelize as a player app to access Logitech Media Server.

**Comment by gravelld** [comment]
Given this definition:

> jukebox mode in the subsonic world means clients just queue songs and these songs are played on the server's audio output.

Logitech Media Server doesn't, I believe, have a jukebox mode. It's just a server, there's no playback functionality. You have to install something like squeezelite to play on the same server as LMS itself.

**Comment by asche77** [comment]
> Given this definition:
> 
> > jukebox mode in the subsonic world means clients just queue songs and these songs are played on the server's audio output.
> 
> Logitech Media Server doesn't, I believe, have a jukebox mode. It's just a server, there's no playback functionality. You have to install something like squeezelite to play on the same server as LMS itself.

That is correct, but on the other hand it's easy to install a player app along with the LMS (server), and that will permit jukebox mode (remote control of music playing on the server audio output).

**Comment by gravelld** [comment]
I don't want to labour the point, but isn't "easy" subjective?

You can say "it's easy to install a player app alongside [server]" for every single server in your list with an "X" next to jukebox.
